src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a id="image" href="/">
+        <img class="images" src="images/logo.png"
+        alt="logo">
+      </a>
```

Comment:
```suggestion
      <a class="image-link" href="/">
        <img 
          class="image"
          src="images/logo.png"
          alt="logo"
        >
      </a>
```
Please, do not use `id` to add styles
Just use class as usually, but different name, there is an example

Also, pay attention to formatting

Reply:
Done, thank you!